besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java

```diff
@@ -266,13 +301,22 @@ public String build() {
       lines.add("Engine JWT: " + engineJwtFilePath);
     }
 
+    lines.add("Using " + txPoolImplementation + " transaction pool implementation");
+
     if (isHighSpec) {
       lines.add("Experimental high spec configuration enabled");
     }
 
-    lines.add("Using " + txPoolImplementation + " transaction pool implementation");
     lines.add("Using " + worldStateUpdateMode + " worldstate update mode");
 
+    if (isTrieLogPruningEnabled) {
+      lines.add("Trie log pruning enabled:");
+      lines.add("  - retention threshold: " + trieLogRetentionThreshold + " blocks");
+      if (trieLogPruningLimit != null) {
+        lines.add("  - prune limit: " + trieLogPruningLimit + " blocks");
+      }
```

Comment:
nit: seems a little verbose to be using three lines in the config logging. can to be shortened to one line?

Reply:
Done this here: https://github.com/hyperledger/besu/pull/6026/commits/6b275220acfe18531b1fcb91e2a5351986679946
Looks like this:
```
# Trie log pruning enabled: retention: 512; prune limit: 30000                                     #
```